config/locales/en.yml

```diff
@@ -2498,6 +2498,7 @@ en:
         cycle_map: Cycle Map
         transport_map: Transport Map
         hot: Humanitarian
+        opnvkarte: ÖPNVKarte
```

Comment:
Shouldn't this be ÖVPNKarte?

Reply:
Isn't that what it is? I don't see a typo

Reply:
Oh you're quite right, it's just very hard to see in this font, at least without my glasses...